runtime-interface/src/gear_sandbox/mod.rs

```diff
@@ -0,0 +1,140 @@
+// This file is part of Gear.
+
+// Copyright (C) Gear Technologies Inc.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+//! Runtime interface for gear node
+
+use sp_runtime_interface::{runtime_interface, Pointer};
+use sp_wasm_interface::HostPointer;
+
+#[cfg(feature = "std")]
+mod detail;
+
+#[cfg(feature = "std")]
+pub use detail::init;
+
+/// Wasm-only interface that provides functions for interacting with the sandbox.
+#[runtime_interface(wasm_only)]
+pub trait Sandbox {
```

Comment:
Consider
```suggestion
pub trait GearSandbox {
```

Reply:
For backward compatibility this need to be named exactly this way